tests/api.devicestatus.test.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var _ = require('lodash');
+var request = require('supertest');
+var should = require('should');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['should' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)